src/pages/ContactUs.tsx

```diff
@@ -53,82 +74,97 @@ const ContactUs = () => {
       });
   };
 
+  const email = (
+    <a href="mailto: support@mediline.io" style={{ textDecoration: "none" }}>
```

Comment:
define the email as a constant at top of file in case we would like to change it later :)

Reply:
Ok, making changes and re-pushing.